DesktopEditor/raster/JBig2/source/LeptonLib/readfile.cpp

```diff
@@ -81,10 +81,10 @@ static const char JP2K_IMAGE_DATA[12] = {
 };
 
 #else
-static const char JP2K_CODESTREAM[4] = { (char)0xff, 0x4f, (char)0xff, 0x51 };
```

Comment:
Just patch line 71:
```
-#if defined(__APPLE__)  // compile on xcode
+#if defined(__clang__)  // compile with clang
```

Reply:
Indeed it's the same result.

Reply:
The clang case should compile with any compilers, not sure why there is this conditional.

Reply:
Could you replace it with `#if` for FreeBSD, our dev team thinks this line should not be changed

Reply:
hmmm in fact it's a patch error! I don't understand why it has been applied since my patch was to cast these numbers rather than removing these casts. It seems that casts have been added in the develop branch! 
I delete this patch.